### DISCUSSION TITLE
flux and OCI

### DISCUSSION THREAD

**Discussion by mobsit** [fluxcd/flux2#3946] [Q&A]
I'm new to using flux, I have experienced  it with a git repository in gitlab and github and it works but now I want to use it with a git repository in Oracle Cloud and I don't know how can I achieve that

**Comment by makkes** [comment]
Welcome to the Flux community! The easiest way to get started with a provider other than GitHub or GitLab is to use `flux bootstrap git`. See the [installation guide](https://fluxcd.io/flux/installation/#generic-git-server) on details how to use it.
